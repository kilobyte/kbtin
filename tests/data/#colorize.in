#colorize x 4/6 foo bar
#sub ~ `
#dosub x {$x}
#show $x
