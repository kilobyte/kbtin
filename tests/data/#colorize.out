`4`f`6`o`4`o`6` `4`b`6`a`4`r`7`
